test/replicate.test.js

```diff
@@ -60,6 +61,13 @@ test('replicate group members', async t => {
   }
 
   function testPersistence () {
+    // TODO: remove this when we re-add member list persistence
```

Comment:
```suggestion
    // NOTE: Currently we do not persist membership because it's simpler.
    // If we decide to re-instate this, or re-instate initialization which loads state into memory
    // (another solution to the same problem) then this code below will be useful and the `t.pass()` 
    // and early `return` should be reverted.
    // TODO: remove this when we re-add member list persistence
    // 
```